Makefile

```diff
@@ -1,23 +1,7 @@
-PKGNAME = climakitae
```

Comment:
Looks like you overwrote the `Makefile` by accident, could you roll this back please?

Reply:
Whoops. Fixed.